interfaces/nginx_route/v0/schemas/requirer.json

```diff
@@ -22,155 +21,142 @@
     "properties": {
         "service-hostname": {
             "$id": "#/properties/service-hostname",
-            "type": "string",
+            "type": "hostname",
```

Comment:
I think the `type` would still be `string`, there would just be a `format` of `hostname`. Anyway, since this will be changed to pydantic its a mute point